protobuf/answer.proto

```diff
@@ -24,25 +24,22 @@ import "protobuf/concept.proto";
 
 package grakn.protocol;
 
-message Answer {
-    oneof answer {
-        AnswerGroup answer_group = 1;
-        ConceptMap concept_map = 2;
-        Number number = 3;
-    }
-}
-
-message AnswerGroup {
-    Concept owner = 1;
-    repeated Answer answers = 2;
-}
-
 message ConceptMap {
     map<string, Concept> map = 1;
     string pattern = 2;
 }
 
+message ConceptMapGroup {
+    Concept owner = 1;
+    repeated ConceptMap concept_map = 2;
```

Comment:
Let's make sure all our `repeated` fields have plural names- this is particularly beneficial to client Python, which uses the field names themselves as property names:
```py
answers: list = match_res.answers
```
is more readable than
```py
answers: list = match_res.answer
```
```suggestion
    repeated ConceptMap concept_maps = 2;
```

Reply:
Fixed in https://github.com/graknlabs/protocol/pull/100/commits/8e16eed30e9a1221201d3b4ff85108dd6874ea86